tests/rptest/services/admin.py

```diff
@@ -187,6 +187,19 @@ def get_partitions(self,
             path = f"{path}/{namespace}/{topic}/{partition}"
         return self._request('get', path, node=node).json()
 
+    def get_transactions(self,
+                         topic=None,
+                         partition=None,
+                         namespace="kafka",
+                         node=None):
+        """
+        Get transaction for current partition
+        """
+        path = "partitions"
+        if topic:
```

Comment:
from what I can tell, if `topic` is not specified and the path is `partitions` then this method--`get_transactions`--won't return transactions at all, but rather, probably generic information about partitions. is that right? if so, then I think this method should be changed to have a consistent behavior of only returning transactions, which probably means that topic,namespace,partitions should not be optional.